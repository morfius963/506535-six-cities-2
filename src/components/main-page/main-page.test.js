import React from "react";
import renderer from "react-test-renderer";
import MainPage from "./main-page.jsx";
import OffersList from "../offers-list/offers-list.jsx";
import fixtureData from "../../__mocks__/offers.js";

jest.mock(`../offers-list/offers-list.jsx`, () => jest.fn().mockReturnValue(null));

describe(`snapshot test`, () => {
  it(`App correctly renders`, () => {
    const tree = renderer
      .create(
          <MainPage
            places = {fixtureData}
          />
      )
      .toJSON();
    expect(OffersList).toHaveBeenCalled();
    expect(tree).toMatchSnapshot();
  });
});
