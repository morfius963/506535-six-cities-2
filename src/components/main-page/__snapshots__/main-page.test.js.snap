// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test App correctly renders 1`] = `
<div
  className="page page--gray page--main"
>
  <header
    className="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        >
          <a
            className="header__logo-link header__logo-link--active"
          >
            <img
              alt="6 cities logo"
              className="header__logo"
              height="41"
              src="img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          className="header__nav"
        >
          <ul
            className="header__nav-list"
          >
            <li
              className="header__nav-item user"
            >
              <a
                className="header__nav-link header__nav-link--profile"
                href="#"
              >
                <div
                  className="header__avatar-wrapper user__avatar-wrapper"
                />
                <span
                  className="header__user-name user__name"
                >
                  Oliver.conner@gmail.com
                </span>
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
  <main
    className="page__main page__main--index"
  >
    <h1
      className="visually-hidden"
    >
      Cities
    </h1>
    <div
      className="tabs"
    />
  </main>
</div>
`;
